index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Mfadhili Gitau</title>
+    <link rel="stylesheet" hrefv="./css/main.css">
+</head>
+<body>
+    <header>
+        <img src="images/menu.svg" alt="" class="menu">
+
```

Comment:
Kindly endeavour to specify appropriate `alt` attributes for your images. This is what the user sees whenever the image happens not to load properly.